src/completions/Tab.ts

```diff
@@ -38,18 +38,17 @@ class BufferCompletionOption extends Completions.CompletionOptionHTML
         const favIconUrl = tab.favIconUrl
             ? tab.favIconUrl
             : Completions.DEFAULT_FAVICON
-        this.html = html`<tr class="BufferCompletionOption option container_${container.color} container_${container.icon} container_${container.name}"
-            >
-                <td class="prefix">${pre.padEnd(2)}</td>
-                <td class="container"></td>
-                <td class="icon"><img src="${favIconUrl}" /></td>
-                <td class="title">${tab.index + 1}: ${tab.title}</td>
-                <td class="content">
-                    <a class="url" target="_blank" href=${tab.url}
-                        >${tab.url}</a
-                    >
-                </td>
-            </tr>`
+        this.html = html`<tr
```

Comment:
@glacambre is this the prettier bug?

Reply:
I could be wrong but I don't think so - as far as I can tell, the newlines/spaces that were added here do not have an impact on the semantics of the template.